src/qt/transactiondesc.cpp

```diff
@@ -299,13 +299,13 @@ QString TransactionDesc::toHTML(CWallet *wallet, CWalletTx &wtx)
 		//Decrypt any embedded messages
 		std::string eGRCMessage = ExtractXML(wtx.hashBoinc,"<MESSAGE>","</MESSAGE>");
 		std::string sOptionsNarr = ExtractXML(wtx.hashBoinc,"<NARR>","</NARR>");
-		std::string sGRCMessage = AdvancedDecrypt(eGRCMessage);
+        //std::string sGRCMessage = AdvancedDecrypt(eGRCMessage);
```

Comment:
ill delete the commented lines on both sections. no sense leaving only one commented out one in a section.